_meta/beat.reference.yml

```diff
@@ -49,6 +49,14 @@ apm-server:
     # If the regexp matches, the stacktrace frame is considered to be a library frame.
     #library_pattern: "node_modules|bower_components|~"
 
+    # Regexp to be matched against a stacktrace frame's `file_name`.
+    # If the regexp matches, the stacktrace frame is not used for calculating error groups.
+    # The default pattern excludes stacktrace frames that have
+    # - an empty filename,
+    # - a filename starting with '/webpack'
+    # - a filename in the format of an absolute path only including the filename, e.g. /myfile.
+    #exclude_from_grouping: "^\\s*$|^/webpack|^[/][^/]*$"
```

Comment:
The `/webpack` part of this pattern looks fine. I understand the rest is something we have from the original Opbeat code (`filename.startswith('/' + frame['filename'].split('/')[-1])`). Do we know why it's like that? If not, I suggest we remove it for now (remove the `|^[/][^/]*` part) and add it back in if we come across the use-case.

Reply:
great, I will change that.

Reply:
@roncohen do you mean to only remove `|^[/][^/]*` or also the part describing the empty filename `^\\s*$|`?

Reply:
I don't know where the empty file pattern part came from. I don't recall seeing that anywhere?

Reply:
I was also part of the original Opbeat logic, I can also remove it, just asking because you explicitly mentioned to remove `|^[/][^/]* `.